cpp/include/raft/sparse/solver/lanczos_types.hpp

```diff
@@ -20,6 +20,12 @@
 
 namespace raft::sparse::solver {
 
+// LA: Largest (algebraic) eigenvalues
```

Comment:
Can this be in doxygen format? Right now checking the doxygen-generated doc this doesn't appear.

Reply:
+1